sale_lead_time/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2025 Quartile
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    factor_warehouse = fields.Float(default=1.0)
```

Comment:
Those fields should be renamed. `factor` of what ?

If it is related to lead time, `lead_time_profile_warehouse_factor` is better .